### ISSUE TITLE
Add wrapper script that automatically picks a Ribosome flavor based on the input

### ISSUE THREAD

**Issue by JesseTG** [sustrik/ribosome#50]
That way, build processes don't have to depend on *which* flavor of Ribosome is used.

**Comment by sustrik** [comment]
Good idea. Can be done in shell. The caveat being that it won't work on Windows.

**Comment by JesseTG** [comment]
Why not a Batch version, too?

**Comment by sustrik** [comment]
The ugly thing about it is that then you add with 2 scripts instead of 3. Somehow better, but it's not clear whether it's worth the additional complexity. Stackoverflow suggest couple of ways to make a file be both shell sctipt and windows batch file, but all of them look like quite a nasty hacks :(

**Comment by JesseTG** [comment]
"Add with 2 scripts"?  I don't follow.

**Comment by sustrik** [comment]
I meant ribosome.bat and ribosome.sh instead of ribosome.js ribosome.py and ribosome.rb. (Wording fixed in the comment above.)
